README.md

```diff
@@ -1,104 +1,181 @@
+> **Note:**
+>
+> This is the README of the v2 Node client. If you are looking for the README of v1 you should look [here](https://github.com/mollie/mollie-api-node/blob/b5873dffdfc84fd8a9347cfda623fea78497405b/README.md).
+
 ![Mollie](https://www.mollie.nl/files/Mollie-Logo-Style-Small.png)
 
-# Mollie API client for Node #
+Node client for Mollie's API.
+
+# About
+
+[Mollie](https://www.mollie.com/) builds payment products, commerce solutions and APIs that let you accept online and mobile payments, for small online stores and Fortune 500s alike. Accepting [iDEAL](https://www.mollie.com/payments/ideal), [Bancontact/Mister Cash](https://www.mollie.com/payments/bancontact), [SOFORT Banking](https://www.mollie.com/payments/sofort), [Credit Card](https://www.mollie.com/payments/credit-card), [SEPA Bank transfer](https://www.mollie.com/payments/bank-transfer), [SEPA Direct debit](https://www.mollie.com/payments/direct-debit), [Bitcoin](https://www.mollie.com/payments/bitcoin), [PayPal](https://www.mollie.com/payments/paypal), [Belfius Direct Net](https://www.mollie.com/payments/belfius), [paysafecard](https://www.mollie.com/payments/paysafecard), [Gift Cards](https://www.mollie.com/payments/gift-cards), and [ING Home’Pay](https://www.mollie.com/payments/ing-homepay), [Giropay](https://www.mollie.com/en/payments/giropay/) and [EPS](https://www.mollie.com/en/payments/eps/) online payments without fixed monthly costs or any punishing registration procedures. Just use the Mollie API to receive payments directly on your website or easily refund transactions to your customers.
+
+## Features
+
+- [Payments](https://www.mollie.com/en/docs/reference/payments/create): are the heart of the Mollie API: this is where most implementations start off. 
```

Comment:
Can you change this to the new v2 endpoints?

Reply:
These URLs always link to the latest version of the API docs (v2 in this case). I'd prefer to link to the docs using the more generic URL's instead the specific ones (which point directly to https://docs.mollie.com).

Reply:
But the client 2.0 will use the v2 API. Why not link to the v2 docs?

Reply:
You're right, changed it 👍